### PR TITLE
Fix 4161 tooltips and show card preview implementation

### PR DESCRIPTION
**Pull request by MV88**
## Description
- tooltips added (also for settings)
- show hide, card preview implemented

## Issues
 - #4161 

**Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines: https://github.com/geosolutions-it/MapStore2/blob/master/CONTRIBUTING.md
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x", remove the others)

- [x] Feature

**What is the current behavior?** (You can also link to an open issue here)
see issue

**What is the new behavior?**
- tooltips are visible in the builder (left)
- the show hide card preview button works

**Does this PR introduce a breaking change?** (check one with "x", remove the other)

 - [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: web/client/translations/data.en-US [hunk 1]
```diff
@@ -1889,6 +1889,39 @@
                 "editMedia": "Change media source",
                 "remove": "Remove"
             },
+            "builder": {
+                "defaults": {
+                    "htmlTitle": "<h1 style='text-align:center'>Insertar título</h1><p style='text-align:center'><em>sottotitolo</em></p>",
```
Comment (offtherailz) [inline web/client/translations/data.en-US]: mixed italian and spanish

File: web/client/translations/data.hr-HR [hunk 2]
```diff
@@ -1888,6 +1888,39 @@
                 "fit": "Fit the container space",
                 "editMedia": "Change media source"
             },
+            "builder": {
```
Comment (offtherailz) [inline web/client/translations/data.hr-HR]: Last time we discussed it we decided that populating with english these files was a bad practice
Did something changed I don't know?

File: web/client/translations/data.en-US [hunk 3]
```diff
@@ -1889,6 +1889,39 @@
...
+                    "htmlSample": "<p>Inserisci del testo qui...</p>",
+                    "titleTitle": "Title Section",
```
Comment (offtherailz) [inline web/client/translations/data.en-US]: We should check if we should add "section" on every title. I think we shouldn't

Comment (MV88) [reply web/client/translations/data.en-US]: mmmh, as far as i know at the moment we have only sections of Titles, not sure if in the future we will have somethign else.. anyway it's a default value which should be similar to the type?

File: web/client/translations/data.hr-HR (hunk 2 above)
Comment (MV88) [reply web/client/translations/data.hr-HR]: ok, sorry, i'll change this in the future..

**Pull request merged**